pkg/virt-controller/watch/migration.go

```diff
@@ -1420,6 +1433,32 @@ func (c *MigrationController) updatePod(old, cur interface{}) {
 	return
 }
 
+// When a resourceQuota is updated, figure out if there are pending migration in the namespace
+// if there are we sould push them into the queue to accelerate the target creation process
+func (c *MigrationController) updateResourceQuota(old, cur interface{}) {
```

Comment:
This is needed since we have an exponential backoff for failed migrations. Knowing explicitly that the failure was due to quota, which has been updated now, would speed up the migration.